src/traces/ohlc/attributes.js

```diff
@@ -102,7 +102,7 @@ module.exports = {
         valType: 'number',
         min: 0,
         max: 0.5,
-        dflt: 0.05,
+        dflt: 0.1,
```

Comment:
at `0.1`

![image](https://cloud.githubusercontent.com/assets/6675409/19326023/c36b5fa8-9096-11e6-8348-49e4c4cb8f63.png)

at `0.3`

![image](https://cloud.githubusercontent.com/assets/6675409/19326035/d0609732-9096-11e6-84bd-d841186efb52.png)

at `0.2`

![image](https://cloud.githubusercontent.com/assets/6675409/19326050/e65dacb4-9096-11e6-9ab3-d7f50b1e446c.png)


I'd vote for `0.2`. 

@chriddyp @cldougl any objections?